src/Components/Footer/style.ts

```diff
@@ -17,24 +17,48 @@ export const Heading = styled.h6`
   text-transform: uppercase;
   font-weight: bold;
   margin-bottom: 4px;
-  margin-left: 130px;
+  margin-left: 90px;
 
   @media only screen and (max-width: 768px) {
     padding-right: 130px;
     margin-top: -50px;
+    text-align: center;
+  }
+
+  @media only screen and (min-width: 769px) {
+    text-align: center;
   }
 `;
 
 export const ContactStyle = styled.div`
   margin-top: 80px;
-  margin-left: 120px;
+  margin-left: 50px;
   float: right;
 
+  div {
+    display: flex;
+    align-items: center;
+    margin-bottom: 10px;
+  }
+
+  i {
+    margin-right: 4px;
+  }
+
+  span {
+    margin-left: 10px;
+    text-align: left;
+  }
+
```

Comment:
why nesting?

Reply:
can i use >(child) selector instead of nesting

Reply:
why do you want to use any kind of nesting?

Reply:
please check latest update I replace nesting with separate style component